lib/catalog/martyrology.ts

```diff
@@ -13,6 +13,18 @@ export const Martyrology: { catalog: MartyrologyCatalog } = {
       name: 'Aaron',
       titles: [Title.Martyr],
     },
```

Comment:
For now, I'm a bit hesitant to add this, as I don't see a clear value in it. Let's keep it for later if it becomes truly necessary.

Reply:
> which i thought was the point of the base catalog?

Yes, exactly, that was the main reason, and from a practical standpoint, it was only to share metadata between different calendars (instead of having to redefine it in each one).

Reply:
While I agree it to leave this outside of the scope of this PR, I disagree that the celebration metadata catalogue (`martyrology.ts`) should include only celebrations inscribed in the Roman Martyrology, as not every person/celebration is inscribed in it, but the celebration might still be inscribed in a particular calendar. Moreover, I consider it as a calendar-agnostic and locale-agnostic source of personal/celebration metadata.

Still, let us leave all of this for a later discussion. :wink:

Reply:
i'll spin up a discussion for it, or loop it in our other martyrology discussion, @tukusejssirs. Could you approve and clear your request for changes so this can be merged after a rebase?

Reply:
Discussion continued here: https://github.com/romcal/romcal/discussions/680